### PR TITLE
Handle no themes issue at install

### LINKED ISSUE #28480: Install cannot run if there is no theme - improve error message
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

If there is no Theme in `themes/` folder you cannot install PrestaShop. When you visit `myshop/install-dev/` you see an error message (see below)

This is OK 👍  it does not make sense to install PrestaShop without a theme. But the error message does not tell you the root issue 😅  so you're stuck. We need to provide an error message that helps the user to understand what is the issue.

### Expected behavior

When there is no Theme in `themes/` folder, we display a message saying that PrestaShop cannot be installed without a valid theme.

### Steps to reproduce

1. Fetch the latest changes from `develop` branch
2. Then remove the `themes/classic` folder (delete it): now there is no valid theme on your shop
3. Try to start the standard web install interface: browse `myshop/install-dev/`
4. See below error

Capture d’écran 2022-05-12 à 18 00 06

### PrestaShop version(s) where the bug happened

develop

### PHP version(s) where the bug happened

_No response_

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | When there is no valid theme available at install display a clear error message that helps user understand the problem (see screenshot)
| Type?             | bug fix
| Category?         | IN
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/28480
| Related PRs       | 
| How to test?      | Please follow the ticket instructions
| Possible impacts? | No

## Screenshot

Capture d’écran 2022-05-12 à 18 04 06

### CODE REVIEW

File: install-dev/init.php [hunk 1]
```diff
@@ -128,6 +128,11 @@
             usort($themes, function ($a, $b) {
                 return strcmp($a, $b);
             });
+
+            if (empty($themes)) {
+                die('Error : Cannot find a valid theme in themes/ folder');
```
Comment (kpodemski) [inline install-dev/init.php]:
```suggestion
                die('Error: Cannot find a valid theme in themes/ folder');
```

Comment (kpodemski) [reply install-dev/init.php]: Ok, nvm, I see it's a standard in this file 😜

**Review by kpodemski — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]
I approve but it needs @Julievrz wording :)

File: install-dev/init.php (hunk 1 above)
Comment (Julievrz) [reply install-dev/init.php]: Well, the standard is a mistake... I totally agree with you, no space in English before colons. 👀

Comment (matks) [reply install-dev/init.php]: Nice spotted @kpodemski and @Julievrz I fixed it

**Review by MeKeyCool — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]
Thanks @matks 
Needs some QA?

**Comment by matks** [comment]
> Needs some QA?

Yes I think 👍 there is a confirmed bug report for QA to use

**Review by florine2623 — APPROVED** [review]
Hello @matks ,

The message is displayed is clear :
Screenshot 2022-06-13 at 09 55 58

I tried deleting the folder themes/classic at any step during the installation, the message is always shown and installation cannot continue. Once I put back the classic folder, installation continues back to from where it stops 👍 

It is QA ✅

**Comment by Progi1984** [comment]
Thanks @matks & @florine2623

**Pull request merged**